packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -419,7 +419,7 @@ Posts.addView("afRecentDiscussionThreadsList", terms => {
 
 Posts.addView("legacyPostUrl", function (terms) {
   return {
-    selector: {"legacyData.url": {$regex: "\/lw\/"+terms.legacyUrlId+"\/.*"}},
+    selector: {"legacyData.url": {$regex: "/lw/"+terms.legacyUrlId+"/.*"}},
```

Comment:
Wait, did you test this? This seems exactly like the kind of thing that could break.

Reply:
I didn't test it on the site, but testing in a JavaScript console:
```
> const terms = { legacyUrlId: "testId" };
  "\/lw\/"+terms.legacyUrlId+"\/.*" === "/lw/"+terms.legacyUrlId+"/.*";
< true
```
The two strings are the same - the backslash escape characters are not included in the actual string that is constructed.